en/simulation/gazebo_worlds.md

```diff
@@ -46,37 +31,30 @@ The source code for supported worlds can be found on GitHub here: [PX4/sitl_gaze
 
 ![McMillan Airfield World](../../assets/simulation/gazebo/worlds/mcmillan_airfield.jpg)
 
-
-## Rubble
-
-[PX4/sitl_gazebo/worlds/rubble.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/rubble.world)
-
 ## Sonoma Raceway
 
-
 [PX4/sitl_gazebo/worlds/sonoma_raceway.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/sonoma_raceway.world)
-
-## Typhoon H480
-
-
-[PX4/sitl_gazebo/worlds/cccc.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/typhoon_h480.world)
-
-## Uneven
-
-[PX4/sitl_gazebo/worlds/uneven.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/uneven.world)
-
-
-## UUV Hippocampus
-
-[PX4/sitl_gazebo/worlds/uuv_hippocampus.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/uuv_hippocampus.world)
+![Sonoma_Raceway](../../assets/simulation/gazebo/worlds/sonoma_raceway.png)
 
 ## Warehouse
 
 [PX4/sitl_gazebo/worlds/warehouse.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/warehouse.world)
+![Warehouse](../../assets/simulation/gazebo/worlds/warehouse.png)
 
 ## Yosemite
 
 [PX4/sitl_gazebo/worlds/yosemite.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/yosemite.world)
 
 ![Yosemite](../../assets/simulation/gazebo/worlds/yosemite.jpg)
 
+## Model Specific Worlds {#model_specific_worlds}
+
+Some [vehicle models](../simulation/gazebo_vehicles.md) rely on the physics / plugins of a specific world.
+The PX4 toolchain will automatically spawn a world that has the same name as the vehicle model if one exists (instead of the default **empty.world**):
+
+The model specific worlds are:
+- [boat.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/boat.world): Includes a surface to simulate buoyancy of the [boat](../simulation/gazebo_vehicles.md#usv).
+- [uuv_hippocampus.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/uuv_hippocampus.world): An empty world used to simulate an underwater environment for the [HippoCampus UUV](../simulation/gazebo_vehicles.md#uuv).
+- [typhoon_h480.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/typhoon_h480.world): Used by [Typhoon H480 (Hexrotor)](../simulation/gazebo_vehicles.md#typhoon_h480) vehicle model.
+  The world includes a gazebo plugin for a simulated camera.
+- [iris_irlock.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/iris_irlock.world): Includes a IR beacon for testing (precision landing](https://docs.px4.io/master/en/advanced_features/precland.html).
```

Comment:
Can you add a section for the associated target in http://dev.px4.io/master/en/simulation/gazebo_vehicles.html ? Also any other missing images/targets that are important?

Reply:
Ah, makes sense. Those are all the model-specific targets so I think it should be good for now